node-training-assignment/src/authentication/auth-controller/login-controller.js

```diff
@@ -0,0 +1,12 @@
+const {findUser} = require('../../user/user-mongo');
+
+exports.loginCintroller = async (req, res) => {
+    const { userName, userPassword } = req.body;
+    const user = await findUser({ userName, userPassword });
```

Comment:
this should go into use-case. As mentioned here https://github.com/RishitaV/Node-Assignment/pull/2#discussion_r1354110269